openmetadata-ui/src/main/resources/ui/src/utils/EntityUtilClassBase.ts

```diff
@@ -11,55 +11,73 @@
  *  limitations under the License.
  */
 import {
-  getContainerDetailPath,
-  getDashboardDetailsPath,
-  getDatabaseDetailsPath,
-  getDatabaseSchemaDetailsPath,
-  getDataModelDetailsPath,
   getEditWebhookPath,
-  getMlModelPath,
-  getPipelineDetailsPath,
+  getEntityDetailsPath,
   getServiceDetailsPath,
-  getStoredProcedureDetailPath,
-  getTableDetailsPath,
-  getTableTabPath,
   getTagsDetailsPath,
-  getTopicDetailsPath,
   getUserPath,
 } from '../constants/constants';
 import { GlobalSettingsMenuCategory } from '../constants/GlobalSettings.constants';
 import { EntityTabs, EntityType } from '../enums/entity.enum';
 import { SearchIndex } from '../enums/search.enum';
 import { getTableFQNFromColumnFQN } from './CommonUtils';
 import {
-  getDataProductsDetailsPath,
   getDomainDetailsPath,
   getGlossaryPath,
   getSettingPath,
   getTeamsWithFqnPath,
 } from './RouterUtils';
-import { getSearchIndexDetailsPath } from './SearchIndexUtils';
 
 class EntityUtilClassBase {
-  public getEntityLink(indexType: string, fullyQualifiedName: string) {
+  public getEntityLink(
+    indexType: string,
```

Comment:
can we use the enum of `SearchIndex` and the  `EntityType` here?

Reply:
Will do that as second phase as it might expand scope of this PR